### DISCUSSION TITLE
Nordictrack x32i Treadmill

### DISCUSSION THREAD

**Discussion by wattsup21** [cagnulein/qdomyos-zwift#2942] [General]
Hello,

I have the treadmill listed above in the title, I went into the treadmill options and did not see it listed as I treadmill so I choose the x22i (same thing but smaller). I paired it to QZ and enter the IP address next to the Nordictrack option, open Zwift, connected to app but running avatar did not move. Anyone experience this or can point me in the right direction.

**Comment by cagnulein** [comment]
Do you see the metrics on qz? Show a debug log https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

**Reply by wattsup21** [reply]
I'll get debug file for you but no metrics were shown on QZ app.

**Reply by cagnulein** [reply]
ok check also the new companion that I just uploaded (3.6.0) and enable the OCR flag. It should fix it in case
Let me know!

**Reply by wattsup21** [reply]
OK, where do I find the OCR flag?

**Reply by wattsup21** [reply]
[debug-Thu_Dec_26_09_47_07_2024_Treadmill.log](https://github.com/user-attachments/files/18260544/debug-Thu_Dec_26_09_47_07_2024_Treadmill.log)



Here is the debug file from yesterday I believe, if this is not enough data, I will do it again. I will enable OCR flag first and I got the new build in test flight you sent out yesterday as well.

**Reply by cagnulein** [reply]
@wattsup21 the ocr flag is in the new companion app 3.6.0 where there is the selection of the device

**Reply by wattsup21** [reply]
Wait, is the new companion app separate from the QZ app?  I cannot find OCR flag, I received a email from you in regard to "new QZ 2.18 (982) update. Sorry for being a pain, lol.

**Reply by cagnulein** [reply]
step back :) is your treadmill comes with an android built in tablet? if so you need to follow this https://github.com/cagnulein/QZCompanionNordictrackTreadmill
